GettingStarted.md

```diff
@@ -0,0 +1,152 @@
+# Introduction
+Welcome to impress.js! This presentation framework allows you to create stunning presentations with the power of CSS3 transformations. It is inspired by the idea behind [prezi.com](https://prezi.com). 
+**NOTE:** This Guide is not made for you, if you have never written HTML and/or CSS before. Knowing your way around in JavaScript certainly helps, but is not a necessity. You may still continue this tutorial and try to understand what we do as you go. 
+
+Also, if you are an experienced (or sorta experienced user, anyway), you might prefer the [DOCUMENTATION](DOCUMENTATION.md).
+
+# Getting started with impress.js
+## Installation / acquiring the framework
+You first need to decide if you want to use the bleeding edge (most up to date, generally very reliable as well, but may contain more problems) or the stable release. You can use one of the methods mentioned below or the direct download link to only the javascript file. The benefit is, you don't have to find the file in the repository and you can copy the link and directly load the file from the cdn. 
+
+### **Direct links to only impress.js file**
+- V2.0.0: https://cdn.jsdelivr.net/gh/impress/impress.js@2.0.0/js/impress.js
+- V1.1.0: https://cdn.jsdelivr.net/gh/impress/impress.js@1.1.0/js/impress.js
+- Source: https://cdn.jsdelivr.net/gh/impress/impress.js/js/impress.js
+
+For older versions, please just replace the version number behind the @!
+
+**Loading the script directly from cdn**
+Loading the script from the cdn is quite straight forward. You need to include a script tag in the head of the HTML document:
+
+```
+<script src="https://cdn.jsdelivr.net/gh/impress/impress.js@2.0.0/js/impress.js"></script>
+```
```

Comment:
This is a great idea.

Except... An important use case for presentations is that you're traveling somewhere and presenting in front of an audience, and no internet.

So... This addition is okay and it simplifies life for many. But we should still make a statement about offline use and self contained presentations. Now... We are back to where we started: Do you write more documentation, or can we somehow simplify impress.js to keep it elegant? It seems the following is possible:

    <script src="https://cdn.jsdelivr.net/gh/impress/impress.js@2.0.0/js/impress.js"></script>
    <script>window.impress || document.write('<script src="js/impress.js">\x3C/script>')</script>

This would first try an online URL for impress.js, but if it didn't load, it fall back to trying a local copy of the same file.

Reply:
Yeah this is just meant for ease of use, if you have WiFi. If you don't, you can obviously just add the js file directly. I personally feel like we should add the js file in the releases tab as a direct download, as this makes the download faster

Reply:
That's a good suggestion...

Reply:
In this section, can you just use the code above. It just works, no need to explain or understand why.